reliquery/storage.py

```diff
@@ -93,6 +101,9 @@ def __init__(self, s3_bucket: str, prefix: str):
 
     def _join_path(self, path: StoragePath) -> str:
         return "/".join([self.prefix] + path)
+    
+    def put_file(self, path: StoragePath, file_path: str) -> None:
+        self.s3.upload_file(file_path, self.s3_bucket,  self._join_path(path))
```

Comment:
the linter may complain about this.

can you try formatting with black?

Reply:
no i didn't do that need to make a habit thanks

Reply:
also done